rllib/BUILD.bazel

```diff
@@ -4320,6 +4316,16 @@ py_test(
     ],
 )
 
+py_test(
+    name = "test_footsies_env",
```

Comment:
maybe: 

`test_footsies_env` -> `examples/envs/classes/multi_agent/footsies/test/footsies_suppress_unity_logs`

to keep it consistent with other names?